lib/composer

```diff
@@ -56,8 +56,9 @@ function install_composer_deps() {
     cp "$target/vendor/composer/bin/composer.phar" "$target/bin/composer.phar"
 
     local required_extensions=$(jq --raw-output '.require // {} | keys | .[]' < "$BUILD_DIR/composer.json" | grep '^ext-' | sed 's/^ext-//')
+    local display_required_extensions=$(echo $required_extensions | tr '\n' ' ')
     if [ -n "$required_extensions" ]; then
-        status "Bundling additional extensions $required_extensions"
+        status "Bundling additional extensions: $display_required_extensions"
```

Comment:
Actually I think we can get rid of this list of extensions as all extension name is listed below when installing it. Don't you think?
```suggestion
        status "Bundling additional extensions
```

Reply:
Yes I agree with you